src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -270,6 +292,47 @@ define(function (require, exports, module) {
         return result.promise();
     }
 
+    /*
```

Comment:
Please use /** instead of /* at every place. JSDoc identifies this. It will help whenever we need to generate documentation for this.